lib/led/matrix.js

```diff
@@ -0,0 +1,13 @@
+var LedControl = require("./ledcontrol"),
+  _ = require("lodash");
+
+// stub implementation; extract functionality from ledcontrol.js
+var Matrix = function(opts) {
+  opts.isMatrix = true;
+  return new LedControl(opts);
+};
```

Comment:
Please use function declarations (here and any other similar use), for consistency.